frontend/src/components/general/BottomNavigation.tsx

```diff
@@ -97,6 +104,7 @@ export default function BottomNavigation({
         {saveAsDraft && (
           <Button
             variant="contained"
+            color="grey"
```

Comment:
I see you added `color=grey` in multiple places. Is that because the style is not recognizing the color?

Reply:
From PR description step 7: In mui 5 all buttons the default grey color has been removed and all buttons have color="primary" by default now. To keep the old colors I applied the changes from the [CodeSandbox demo](https://codesandbox.io/s/mimic-v4-button-default-color-bklx8?file=/src/Demo.tsx) provided in the migration guide. Following the guide I introduced a "grey" color to the button component and applied it on all contained buttons that didn't have a color specified.